protocol/codec_tester.go

```diff
@@ -205,6 +261,11 @@ func EncodingTest(template msgpMarshalUnmarshal) error {
 func RunEncodingTest(t *testing.T, template msgpMarshalUnmarshal) {
 	for i := 0; i < 1000; i++ {
 		err := EncodingTest(template)
+		if err == errSkipRawMsgpTesting {
```

Comment:
Although it's probably too much effort to get consistency between msgp and go-codec for raw fields, it would be nice to make sure that `msgp` is at least self-consistent with respect to raw fields (i.e., forall x, decode(encode(x))=x).

Reply:
nice idea. I will add it in a subsequent PR.